### PR TITLE
Added Deadband to MiniPID class

### PR DESCRIPTION
**Pull request by 2141Robotics**
Added deadband to MiniPID class to make steering smoother and to allow for error and stick drift.

### CODE REVIEW

File: quail/src/main/java/com/mineinjava/quail/util/geometry/AccelerationLimitedVector.java
Comment (Astr0clad) [inline quail/src/main/java/com/mineinjava/quail/util/geometry/AccelerationLimitedVector.java:1]: This file needs to be removed from this PR, it is being PRed in #24

**Review by Astr0clad — CHANGES_REQUESTED** [review]
Resolve fix, test, then it lgtm

File: quail/src/main/java/com/mineinjava/quail/util/geometry/AccelerationLimitedVector.java
Comment (Mineinjava) [reply quail/src/main/java/com/mineinjava/quail/util/geometry/AccelerationLimitedVector.java:1]: fix by deleting #24

**Comment by Mineinjava** [comment]
tested--working

**Comment by Mineinjava** [comment]
fixes #22

**Pull request merged**